handsanitizer/JsonInputParser.cpp

```diff
@@ -62,9 +62,9 @@ namespace handsanitizer {
 
         auto parsedArrayElTypeLvalue = declarationManager->getUniqueTmpCPPVariableNameFor();
         JsonParsingCandidate arrayElTypeCandidate = {.lvalueName = parsedArrayElTypeLvalue,
-                .jsonRoot = candidate.jsonRoot + "[" + iterator_name + "]",
```

Comment:
Same here.

Reply:
For a historic reference, this can't be converted as .jsonRoot contains the name of the identifier to the json object inside the generated CPP code. Which isn't something we can represent with the API.